app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,12 @@
+class FoodsController < ApplicationController
+  def index
+    food_input          = params[:q]
+    if food_input       == ""
+     flash[:alert]      = "Please input a food item to search!"
+     redirect_to root_path
```

Comment:
Love the sad path implementation here. Another protection I usually use is `.nil?`. And, for the comparison you're doing here, it's best to use `.blank?` instead of `== ""`